go.mod

```diff
@@ -1,15 +1,82 @@
 module github.com/axelarnetwork/utils
 
-go 1.16
+go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.44.3
+	github.com/cosmos/cosmos-sdk v0.45.1
+	github.com/matryer/moq v0.2.6
+	github.com/stretchr/testify v1.7.1
+	github.com/tendermint/tendermint v0.34.16
```

Comment:
`cosmos-sdk v0.45.1` uses tendermint `v0.34.14`. `16` just has some bug fixes. Preferences between matching the version uses the latest patch release? I'll use that convention for dependencies in other repos.

Reply:
I think updating patch releases is fine